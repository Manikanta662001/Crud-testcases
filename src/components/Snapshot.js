import React from 'react'

const Snapshot = () => {
  return (
    <div>
        <h1>Snapshot</h1>
        <h1>Snap 2</h1>
        <h3>Hii this is Snapshot page</h3>
        <h4>Adding more content</h4>
        <h4>Adding more content</h4>
        <h6>Snapshot matching</h6>
    </div>
  )
}
export default Snapshot