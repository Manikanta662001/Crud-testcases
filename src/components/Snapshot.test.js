import { render, cleanup } from "@testing-library/react";
import Snapshot from "./Snapshot";

afterEach(cleanup);
describe("SNAPSHOT", () => {
  test("take snapshot and matches with it", () => {
    const { asFragment } = render(<Snapshot />);
    expect(asFragment(<Snapshot />)).toMatchSnapshot();
  });
});

describe("truthy and falsy", () => {
  test("truthy value must be true", () => {
    expect(true).toBe(true);
  });
  test("falsy value must be false", () => {
    expect(false).toBe(false);
  });
});
