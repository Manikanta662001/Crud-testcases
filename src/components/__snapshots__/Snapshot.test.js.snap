// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT take snapshot and matches with it 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Snapshot
    </h1>
    <h1>
      Snap 2
    </h1>
    <h3>
      Hii this is Snapshot page
    </h3>
    <h4>
      Adding more content
    </h4>
    <h4>
      Adding more content
    </h4>
    <h6>
      Snapshot matching
    </h6>
  </div>
</DocumentFragment>
`;
